# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/gnophone/linux-2.6.9-ixjuser_h.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/modules/audio-phone.c.orig	2004-12-02 02:32:42.335616832 +0100
+++ ./src/modules/audio-phone.c	2004-12-02 02:32:59.888948320 +0100
@@ -26,6 +26,7 @@
 #include <sys/ioctl.h>
 #include <linux/telephony.h>
 #ifdef IXJ_ENHANCED
+#include <linux/compiler.h>
 #include <linux/ixjuser.h>
 #endif
 #include <sys/stat.h>
